server/src/core/import_resolver.rs

```diff
@@ -82,7 +85,8 @@ pub fn resolve_from_stmt(session: &mut SessionInfo, source_file_symbol: &Rc<RefC
         level,
         from_stmt);
     drop(_source_file_symbol_lock);
-    let source_path = source_file_symbol.borrow().paths()[0].clone();
+    // source_file_symbol is either a file or a namespace with a single path
+    let source_path = source_file_symbol.borrow().path().as_vec()[0].clone();
```

Comment:
This simply keeps the old behavior. But it feels dodgy. What do think?
See also: [the previous comment](https://github.com/odoo/odoo-ls/pull/484/files#r2550524957)